.travis.yml

```diff
@@ -2,3 +2,19 @@ language: ruby
 cache: bundler
 rvm:
   - 2.5.1
+
+dist: trusty
+sudo: required
+addons:
+  chrome: stable
+
+before_install:
+  - wget https://chromedriver.storage.googleapis.com/2.35/chromedriver_linux64.zip
```

Comment:
why is this? https://docs.travis-ci.com/user/gui-and-headless-browsers/#using-the-chrome-addon-in-the-headless-mode it doesn't say you need to download chrome